src/views/CreateInstance.vue

```diff
@@ -0,0 +1,157 @@
+      :backLink="{ link: '/resources' }"
+    />
+    <ListSection :list="nonAbstractResourceTypesList"/>
+  </main>
+
+  <main v-else-if="formState === 1" class="create-Instance">
+    <ViewHeader title="Choose attribute values for the new Instance" :backLink="{ onClick: previousStep }"/>
+    <ListSection :title="`Attributes of ${resourceType.name}`">
+      <Li v-for="attribute in newResourceInstance.attributes" :key="attribute.name">
+        <TypeSensitiveInput
+          :resourceType="newResourceInstance.resourceType"
+          :resourceInstanceAttribute="attribute"
+        />
+      </Li>
+    </ListSection>
+
+    <Button text="save" :onClick="createResourceInstance"/>
+  </main>
+</template>
+
+<script lang="ts">
+import { Component, Mixins } from 'vue-property-decorator';
+import {
+  ResourceTypes,
+  ResourceType,
+  ResourceTypeAttribute,
+  ResourceInstances,
+} from '@/apis/rembrandt/rembrandt';
+import Li, { ListEntry } from '@/components/Li.vue';
+import ViewHeader from '@/components/ViewHeader.vue';
+import ListSection from '@/components/ListSection.vue';
+import TypeSensitiveInput from '@/components/TypeSensitiveInput.vue';
+import Input from '@/components/Input.vue';
+import Toggle from '@/components/Toggle.vue';
+import Button from '@/components/Button.vue';
+import Select from '@/components/Select.vue';
+import Utils from '@/utils/Utils';
+import Translate from '@/mixins/Translate';
+import { ResourceInstance, ResourceInstanceNullObject } from '@/apis/rembrandt/rembrandt';
+
+@Component({
+  components: {
+    ListSection,
+    Li,
+    ViewHeader,
+    TypeSensitiveInput,
+    Input,
+    Toggle,
+    Button,
+    Select,
+  },
+})
+export default class Types extends Mixins(Translate) {
+  // region public static methods
+  public static emptyResourceType(): ResourceType {
+    return {
+      name: '',
+      abstract: false,
+      attributes: [],
+    };
+  }
+
+  public static emptyResourceTypeAttribute(): ResourceTypeAttribute {
+    return {
+      name: '',
+      dataType: 'string',
+      required: true,
+    };
+  }
+  // endregion
+
+  // region private static methods
+  // endregion
+
+  // region public members
+  public resourceType: ResourceType;
+  public resourceTypes: ResourceType[] = [];
+  public nonAbstractResourceTypes: ResourceType[] = [];
+  public newResourceInstance: ResourceInstance;
+  public typeId: string = '';
+  public formState: number = 0;
+
+  public get nonAbstractResourceTypesList(): ListEntry[] {
+    const nonAbstractResourceTypes =  this.resourceTypes.filter((resourceType) => !resourceType.abstract);
+    return Utils.resourceTypesToList(nonAbstractResourceTypes, this.selectResourceType);
+  }
+
+  // endregion
+
+  // region private members
+  // endregion
+
+  // region constructor
+  constructor() {
+    super();
+    this.newResourceInstance = ResourceInstanceNullObject;
+    this.newResourceInstance.attributes = [];
+    this.resourceType = Types.emptyResourceType();
```

Comment:
`Types.` due to misnamed class?

Reply:
jep, but i changed it to the resourceTypeNullObject anyway